Formula/wrangler.rb

```diff
@@ -13,9 +23,13 @@ class Wrangler < Formula
   depends_on "erlang"
 
   def install
-    ENV.deparallelize
     system "./configure", "--prefix=#{prefix}"
     system "make"
     system "make", "install"
   end
+
+  test do
+    suffixtree = Dir.glob("#{lib}/erlang/*/*/*/suffixtree").shift
+    assert File.executable?(suffixtree), "suffixtree must be executable"
```

Comment:
Do we have anything we can run (even for a `--version`)? I fear this won't detect breakage.